subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BNDSmokeTest.groovy

```diff
@@ -0,0 +1,481 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.smoketests
+
+import org.gradle.integtests.fixtures.ToBeFixedForConfigurationCache
+import org.gradle.util.GradleVersion
+import org.gradle.util.internal.VersionNumber
+import spock.lang.Issue
+
+import java.util.jar.Attributes
+import java.util.jar.JarFile
+
+/**
+ * Smoke tests for <a href="https://github.com/bndtools/bnd/blob/master/gradle-plugins/README.md">the BND plugin</a>.
+ */
+class BNDSmokeTest extends AbstractPluginValidatingSmokeTest {
+    def setup() {
+        settingsFile << """
+pluginManagement {
+    plugins {
+        id "biz.aQute.bnd.builder" version "${TestedVersions.bnd}"
+    }
+}
+    
+rootProject.name = 'bnd-smoke-test'
+"""
+    }
+
+    def "BND plugin writes direct external dependency versions to the manifest"() {
+        given:
+        def commonsVersion = "3.12.0"
+        def calculatedCommonsVersionRange = "[3.12,4)"
+        buildFile << """
+plugins {
+    id "biz.aQute.bnd.builder"
+}
+
+${mavenCentralRepository()}
```

Comment:
Optional suggestion: move this to settings as a centralized dependency declaration so you can drop it in all the tests below.